src/main/java/seedu/address/model/assignment/DueDate.java

```diff
@@ -48,23 +50,27 @@ public DueDate(String date, String time) {
      */
     public DueDate(String date) {
         requireNonNull(date);
-        // TODO: use Validation regex to verify dateTime format
-        //checkArgument(isValidPhone(phone), MESSAGE_CONSTRAINTS);
+        checkArgument(isValidDate(date), MESSAGE_CONSTRAINTS);
         this.date = LocalDate.parse(date, PARSE_DATE_FORMAT);
         this.time = LocalTime.parse("2359", PARSE_TIME_FORMAT);
```

Comment:
Perhaps you might consider creating a constant for ```"2359"```? For example, ```LATEST_TIME_IN_DAY```.

Reply:
Good point!